doc/apidoc/smac.intensification.intensification.rst

```diff
@@ -0,0 +1,7 @@
+smac\.intensification\.intensification module
+=============================================
+
+.. automodule:: smac.intensification.intensification
```

Comment:
Yes, this is intended. It is toggled via [this option](http://www.sphinx-doc.org/en/stable/ext/autodoc.html#confval-autoclass_content). Would you prefer it to be shown?